src/packages/core/src/decorators/relationship-field.ts

```diff
@@ -36,7 +36,7 @@ export function RelationshipField<
 	D extends BaseDataEntity = G['dataEntity']
 >(
 	returnTypeFunc: ReturnTypeFunc,
-	{ relatedField, id, nullable = true }: RelationshipFieldOptions<D>
+	{ relatedField, id, nullable = false }: RelationshipFieldOptions<D>
```

Comment:
This is the correct change as it brings the default inline with the field resolver. However, it is potentially a breaking change.